assets/js/apps/tiles/components/DatasetFilter.jsx

```diff
@@ -18,81 +18,70 @@ const DatasetFilter = ({ onFilter, onSort }) => {
-                    <button className="fa fa-sort-numeric-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datafileCount', 'desc')} />
-                  </div>
+                <div className="col col-md-4">
+                  <button className="fa fa-sort-numeric-asc me-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datafileCount', 'asc')} />
+                  <button className="fa fa-sort-numeric-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datafileCount', 'desc')} />
                 </div>
-                <div className="row">
-                  <div className="col col-md-8">
-                    <span className="dropdown-item" href="#">
-                      Dataset size
-                    </span>
-                  </div>
-                  <div className="col col-md-4">
-                    <button className="fa fa-sort-numeric-asc mr-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetSize', 'asc')} />
-                    <button className="fa fa-sort-numeric-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetSize', 'desc')} />
-                  </div>
+              </div>
+              <div className="row">
+                <div className="col col-md-8">
+                  <span className="dropdown-item" href="#">Size</span>
+                </div>
+                <div className="col col-md-4">
+                  <button className="fa fa-sort-numeric-asc me-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetSize', 'asc')} />
+                  <button className="fa fa-sort-numeric-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetSize', 'desc')} />
+                </div>
+              </div>
+              <div className="row">
+                <div className="col col-md-8">
+                  <span className="dropdown-item" href="#">Last modified</span>
                 </div>
-                <div className="row">
-                  <div className="col col-md-8">
-                    <span className="dropdown-item" href="#">
-                      Dataset modified time
-                    </span>
-                  </div>
-                  <div className="col col-md-4">
-                    <button className="fa fa-sort-numeric-asc mr-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetTime', 'asc')} />
-                    <button className="fa fa-sort-numeric-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetTime', 'desc')} />
-                  </div>
+                <div className="col col-md-4">
+                  <button className="fa fa-sort-numeric-asc me-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetTime', 'asc')} />
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 36 spaces but found 18.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
                                    <button className="fa fa-sort-numeric-asc me-2 mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('datasetTime', 'asc')} />
```